### PR TITLE
Add tftp_host to tftp service

### PR DESCRIPTION
**Pull request by erinix**
Ticket: #75675

### CODE REVIEW

File: src/app/helptext/services/components/service-tftp.ts [hunk 1]
```diff
@@ -11,6 +11,9 @@ tftp_newfiles_placeholder : T('Allow New Files'),
 tftp_newfiles_tooltip : T('Set when network devices need to send files to\
  the system.'),
 
+tftp_host_placeholder: T('Host'),
+tftp_host_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-tftp.ts]:
```suggestion
tftp_host_tooltip: T('The default host to use for TFTP transfers. Enter an IP address. Example: 192.0.2.1'),
```

File: src/app/helptext/services/components/service-tftp.ts [hunk 2]
```diff
@@ -11,6 +11,9 @@ tftp_newfiles_placeholder : T('Allow New Files'),
...
+
 tftp_port_placeholder : T('Port'),
 tftp_port_tooltip : T('Enter a UDP prort to listen for TFTP requests.'),
```
Comment (Mrt134) [inline src/app/helptext/services/components/service-tftp.ts]:
```suggestion
tftp_port_tooltip : T('The UDP port number that listens for TFTP requests. Example: 8050'),
```

File: src/app/helptext/services/components/service-tftp.ts [hunk 3]
```diff
@@ -11,8 +11,11 @@ tftp_newfiles_placeholder : T('Allow New Files'),
 tftp_newfiles_tooltip : T('Set when network devices need to send files to\
  the system.'),
 
+tftp_host_placeholder: T('Host'),
+tftp_host_tooltip: T('The default host to use for TFTP transfers. Enter an IP address. Example: 192.0.2.1'),
+
 tftp_port_placeholder : T('Port'),
-tftp_port_tooltip : T('Enter a UDP prort to listen for TFTP requests.'),
+tftp_port_tooltip : T('The UDP port number that listens for TFTP requests. Example: 8050'),
```
Comment (wblock) [inline src/app/helptext/services/components/service-tftp.ts]: 'Port number on which the TFTP service listens. Example: <i>8050</i>.'

Comment (erinix) [reply src/app/helptext/services/components/service-tftp.ts]: I believe these were changed in a different PR which was also merged, please look at what is currently in frenas/11.2-stable

**Pull request merged**